po/cs_CZ.UTF-8.po

```diff
@@ -0,0 +1,268 @@
+msgstr "Povolit přístup ke schránce"
+
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-read.blp:7
+msgid ""
+"An application is attempting to read from the clipboard. The current "
+"clipboard contents are shown below."
+msgstr ""
+"Aplikace se snaží přečíst obsah ve schránce. Obsah schránky "
+"je zobrazen dole."
+
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-read.blp:10
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-write.blp:10
+msgid "Deny"
+msgstr "Zakázat"
+
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-read.blp:11
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-write.blp:11
+msgid "Allow"
+msgstr "Povolit"
+
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-write.blp:7
+msgid ""
+"An application is attempting to write to the clipboard. The current "
+"clipboard contents are shown below."
+msgstr ""
+"Aplikace se snaží zapsat do schránky. Obsah schránky "
+"je zobrazen dole."
+
+#: src/apprt/gtk/ui/1.5/ccw-paste.blp:6
+msgid "Warning: Potentially Unsafe Paste"
+msgstr "Varování: Potenciálně nebezpečné vložení"
+
+#: src/apprt/gtk/ui/1.5/ccw-paste.blp:7
+msgid ""
+"Pasting this text into the terminal may be dangerous as it looks like some "
+"commands may be executed."
+msgstr ""
+"Vložení tohoto textu do terminálu může být nebezpečné protože to vypadá že "
+"nějáké příkazy mohou být spuštěny."
+
+#: src/apprt/gtk/inspector.zig:144
+msgid "Ghostty: Terminal Inspector"
+msgstr "Ghostty: Inspektor terminálu"
+
+#: src/apprt/gtk/Surface.zig:1243
+msgid "Copied to clipboard"
+msgstr "Zkopírováno do schránky"
+
+#: src/apprt/gtk/CloseDialog.zig:47
+msgid "Close"
+msgstr "Zavřít"
+
+#: src/apprt/gtk/CloseDialog.zig:87
+msgid "Quit Ghostty?"
+msgstr "Zavřít Ghostty?"
+
+#: src/apprt/gtk/CloseDialog.zig:88
+msgid "Close Window?"
+msgstr "Zavřít okno?"
+
+#: src/apprt/gtk/CloseDialog.zig:89
+msgid "Close Tab?"
+msgstr "Zavřít kartu?"
+
+#: src/apprt/gtk/CloseDialog.zig:90
+msgid "Close Split?"
+msgstr "Zavřít rozdělení?"
+
+#: src/apprt/gtk/CloseDialog.zig:96
+msgid "All terminal sessions will be terminated."
+msgstr "Všechny procesy budou terminovány."
+
+#: src/apprt/gtk/CloseDialog.zig:97
+msgid "All terminal sessions in this window will be terminated."
+msgstr "Všechny procesy v tomto okně budou terminovány."
+
+#: src/apprt/gtk/CloseDialog.zig:98
+msgid "All terminal sessions in this tab will be terminated."
+msgstr "Všechny procesy v této kartě budou terminovány."
+
+#: src/apprt/gtk/CloseDialog.zig:99
+msgid "The currently running process in this split will be terminated."
+msgstr "Momentálně běžící proces v tomto rozdělení bude terminován."
+
+#: src/apprt/gtk/Window.zig:200
+msgid "Main Menu"
+msgstr "Hlavní menu"
+
+#: src/apprt/gtk/Window.zig:221
+msgid "View Open Tabs"
+msgstr "Zobrazit otevřené karty"
+
+#: src/apprt/gtk/Window.zig:295
+msgid ""
+"⚠️ You're running a debug build of Ghostty! Performance will be degraded."
+msgstr ""
+"⚠️ Používáte testovací verzi Ghostty! Výkon bude snížen."
+
+#: src/apprt/gtk/Window.zig:725
+msgid "Reloaded the configuration"
+msgstr "Konfigurace byla aktualizována"
```

Comment:
This is a bit misleading. Reloaded is never translated as `aktualizováno`.  The translation would be "Konfigurace byla znovu načtěna" or "Konfigurace byla obnovena". I prefer the first variant.